### PR TITLE
[Fix] Now users will see accordion selection on focus

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://weboo.atlassian.net/browse/OD-27?atlOrigin=eyJpIjoiNjJhMTg3YWU0YjZkNDY2OGE5YzRjNTBkNjY4MDAzYTMiLCJwIjoiaiJ9

## Description
Added z index for the focused accordion to show box-shadow.

## Screenshots/screencasts
https://share.getcloudapp.com/eDuBZAnW

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: css/build.css [hunk 1]
```diff
@@ -125,4 +125,12 @@ script {
   -webkit-transform: none;
   -ms-transform: none;
   transform: none;
-}
\ No newline at end of file
+}
+
+.panel.panel-default.focus-outline {
```
Comment (tonytlwu) [inline css/build.css]: Under what scenario will the `focus-outline` be missing?

Comment (YaroslavOvdii) [reply css/build.css]: @tonytlwu So far I can't see such scenarios. Should I remove `focus-outline` from the selector?

Comment (tonytlwu) [reply css/build.css]: Yes please

Comment (YaroslavOvdii) [reply css/build.css]: @tonytlwu removed it.

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: All accordions
Verify: Ensure accordion layouts and positioning in relation to other elements are correct

**Pull request merged**